webpack.config.js

```diff
@@ -94,7 +94,11 @@ module.exports = {
     filename: '[name]/index.js',
   },
   plugins: [
-    new LodashModuleReplacementPlugin({ shorthands: true }),
+    new LodashModuleReplacementPlugin({
+      shorthands: true,
+      collections: true,
+      paths: true,
+    }),
```

Comment:
What's going on here?

Reply:
[lodash-webpack-plugin](https://github.com/lodash/lodash-webpack-plugin) disables certain features to create smaller builds. I needed this particular feature when using `_.get` in `$t`.